.gitmodules

```diff
@@ -1197,3 +1197,6 @@
 [submodule "vendor/grammars/zephir-sublime"]
 	path = vendor/grammars/zephir-sublime
 	url = https://github.com/phalcon/zephir-sublime
+[submodule "vendor/grammars/learnPortugol"]
+	path = vendor/grammars/learnPortugol
+	url = https://github.com/luisgbr1el/learnPortugol.git
```

Comment:
This grammar doesn't have a license. We can't accept grammars without licenses. The license also needs to be also be one of these: https://github.com/github/linguist/blob/257425141d4e2a5232786bf0b13c901ada075f93/vendor/licenses/config.yml#L2-L11

The grammar in this repo will also not be found by Linguist because it doesn't appear in one of the locations expected for this type of grammar file:

https://github.com/github/linguist/blob/bd944c3e80b7e8d854c6e5d6a04bf4c336dda682/tools/grammars/compiler/loader.go#L125